src/index.ts

```diff
@@ -131,7 +131,7 @@ async function handleUpload(
     if (
         file.url.includes(`/${config.containerName}/`) &&
         config.removeCN &&
-        config.removeCN == 'true'
+        config.removeCN !== 'false'
```

Comment:
It could be a better approach to compare the value with the string "true" to avoid typo-caused misconfigurations, like: `config.removeCN === 'true'`
Btw, thanks for the fix! 🎖️

Reply:
+1 would prefer `config.removeCN === 'true'`